apps/ledger-live-mobile/src/families/near/StakingFlow/02-Summary.tsx

```diff
@@ -0,0 +1,494 @@
+            validator={chosenValidator}
+            account={account}
+            amount={transaction.amount}
+          />
+        </View>
+      </View>
+      <View style={styles.footer}>
+        <LText
+          style={[styles.fieldStatus]}
+          color={error ? "alert" : warning ? "orange" : "darkBlue"}
+          numberOfLines={2}
+        >
+          <TranslatedError error={error || warning} />
+        </LText>
+        <Button
+          event="SummaryContinue"
+          type="primary"
+          title={<Trans i18nKey="common.continue" />}
+          containerStyle={styles.continueButton}
+          onPress={onContinue}
+          disabled={bridgePending || !!bridgeError || hasErrors}
+          pending={bridgePending}
+        />
+      </View>
+    </SafeAreaView>
+  );
+}
+
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    flexDirection: "column",
+  },
+  body: {
+    flex: 1,
+    paddingHorizontal: 16,
+    justifyContent: "space-around",
+  },
+  validatorCircle: {
+    borderWidth: 1,
+    borderStyle: "dashed",
+  },
+  changeValidator: {
+    position: "absolute",
+    right: -4,
+    top: -4,
+  },
+  delegatingAccount: {
+    paddingTop: 26,
+    alignItems: "center",
+  },
+  accountBalanceTag: {
+    marginTop: 8,
+    borderRadius: 4,
+    padding: 4,
+    alignItems: "center",
+  },
+  accountBalanceTagText: {
+    fontSize: 11,
+  },
+  accountName: {
+    maxWidth: 180,
+  },
+  summary: {
+    alignItems: "center",
+    marginVertical: 30,
+  },
+  summaryLine: {
+    marginVertical: 10,
+    flexDirection: "row",
+    height: 40,
+    alignItems: "center",
+  },
+  summaryWords: {
+    marginRight: 6,
+    fontSize: 18,
+  },
+  validatorSelection: {
+    flexDirection: "row",
+    alignItems: "center",
+    borderRadius: 4,
+    height: 40,
+  },
+  validatorSelectionText: {
+    paddingHorizontal: 8,
+    fontSize: 18,
+    maxWidth: 240,
+  },
+  validatorSelectionIcon: {
+    borderTopRightRadius: 4,
+    borderBottomRightRadius: 4,
+    alignItems: "center",
+    justifyContent: "center",
+    width: 32,
+    height: 40,
+  },
+  footer: {
+    flexDirection: "column",
+    alignItems: "center",
+    paddingHorizontal: 16,
+    paddingTop: 24,
+    paddingBottom: 16,
+  },
+  continueButton: {
+    alignSelf: "stretch",
+    marginTop: 12,
+  },
+  fieldStatus: {
+    fontSize: 14,
+    textAlign: "center",
+  },
+});
+
+function SummaryWords({
+  validator,
+  account,
+  amount,
+  onChangeValidator,
+  onChangeAmount,
+}: {
+  validator?: NearValidatorItem;
+  account: AccountLike;
+  amount: BigNumber;
+  onChangeValidator: () => void;
+  onChangeAmount: () => void;
+}) {
+  const unit = getAccountUnit(account);
+  const formattedAmount = formatCurrencyUnit(unit, amount, {
+    disableRounding: true,
+    alwaysShowSign: false,
+    showCode: true,
+  });
+  return (
+    <>
+      <Line>
+        <Words>
+          <Trans i18nKey={"near.staking.iStake"} />
```

Comment:
Good catch - I see that the `react/jsx-curly-brace-presence` ESLint rule is turned off in LLM, if it could be turned back on then this would be caught going forward.